Consistent fly.io and no "Fly App" (#909)
superfly/docs: app-guides/fauna.html.md

## Starter kit

> We've provided [starter kits](https://github.com/orgs/fauna-labs/repositories?q=fly-io-starter) to help you launch, as quickly as possible, a Fly.io app that uses Fauna as the database backend. Before you start, read the short section below to learn how Fauna and Fly.io makes for a great combination when building low latency, dynamic global applications.  

Author: ```suggestion
Fauna provides [starter kits](https://github.com/orgs/fauna-labs/repositories?q=fly-io-starter) to help you quickly launch an app on Fly.io that uses Fauna as the database backend. Before you start, read the short section below to learn how Fauna and Fly.io makes for a great combination when building low latency, dynamic global applications.  
```
